ansible/roles/base/tasks/main.yml

```diff
@@ -17,18 +15,23 @@
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
-      apt: name=python-pip
+      apt: 
```

Comment:
I downloaded pyCharm will be using that for my changes now, lets see if it helps :)